creators/accra/trips_creator_accra.py

```diff
@@ -18,7 +19,7 @@ def add_trips_to_schedule(self, schedule, data):
 
         lines = data.routes
         for route_ref, line in sorted(lines.iteritems()):
-            if type(line).__name__ != "RouteMaster":
+            if not isinstance(line, Line):
```

Comment:
Much better! 👍

Reply:
Thanks for the first quick feedback. I pushed a new commit with fixes but sadly it does not show a cool diff between the different versions of the pull request like in GitLab 🙁